Improve log line for `routing: not found`

{"time":"2023-09-25T11:25:36.491248447Z","level":"ERROR","msg":"routing: not found","address":"0x89825D58A7E2C198a06125be9CD0631317f9A07B"}
panic: routing: not found

--- a/node/engine/messageservice/p2p-message-service/service.go
+++ b/node/engine/messageservice/p2p-message-service/service.go
@@ -290,7 +290,7 @@ func (ms *P2PMessageService) Send(msg protocols.Message) error {
 		ms.logger.Warn("did not find scAddr in local peers map, fetching from DHT", "scAddr", msg.To.String())
 		peerId, err = ms.getPeerIdFromDht(msg.To.String())
 		if err != nil {
-			ms.logger.Error("did not find scAdr in DHT", "scAddr", msg.To.String())
+			ms.logger.Error("did not find scAddr in DHT", "scAddr", msg.To.String())
 			return err
 		}
 	} else {

--- a/node/engine/messageservice/p2p-message-service/service.go
+++ b/node/engine/messageservice/p2p-message-service/service.go
@@ -287,8 +287,10 @@ func (ms *P2PMessageService) Send(msg protocols.Message) error {
 	// query the dht to retrieve the peerId, then store in local map for next time
 	peerId, ok := ms.peers.Load(msg.To.String())
 	if !ok {
+		ms.logger.Warn("did not find scAddr in local peers map, fetching from DHT", "scAddr", msg.To.String())
 		peerId, err = ms.getPeerIdFromDht(msg.To.String())
 		if err != nil {
+			ms.logger.Error("did not find scAdr in DHT", "scAddr", msg.To.String())
 			return err
 		}
 	} else {

